components/skus/browser/rs/lib/src/storage/kv.rs

```diff
@@ -162,6 +195,38 @@ where
         store.set_state(&state)
     }
 
+    #[instrument]
+    async fn get_time_limited_v2_creds(
+        &self,
+        item_id: &str,
+    ) -> Result<Option<TimeLimitedV2Credentials>, InternalError> {
+        let mut store = self.get_store()?;
+        let mut state: KVState = store.get_state()?;
+
+        if state.credentials.is_none() {
+            state.credentials = Some(CredentialsState { items: HashMap::new() });
```

Comment:
looks like `#[derive(Default)]` on `CredentialsState` might be useful for `Some(CredentialsState::default())` here and below

Reply:
https://github.com/brave/brave-core/pull/15465/commits/9c9188c99dcc3e8bb08fd4835f602a62155975ae